ably/realtime/connection.py

```diff
@@ -150,6 +174,8 @@ def try_connect(self):
 
     async def _connect(self):
         if self.__state == ConnectionState.CONNECTED:
+            if self.__ttl_task:
```

Comment:
This looks wrong to me - I think we should cancel the task only once we receive a `CONNECTED` message from the service. Having this code here means the task will be cancelled if the user calls `Realtime.connect()` while already connected and the client will never transition to `SUSPENDED` if they subsequently lose connection

Reply:
I'll take this out, seems redundant. I still think if they lose connection, there will be a state change to `DISCONNECTED`, and that should trigger the connection state task.